projects/offline_mapping/build_map.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env python
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import click
+import glob
+import pickle
+import sys
+import torch
+from pathlib import Path
+import numpy as np
+from PIL import Image
+import matplotlib.pyplot as plt
+
+# TODO Install home_robot and remove this
+sys.path.insert(
+    0,
+    str(Path(__file__).resolve().parent.parent.parent / "src/home_robot"),
+)
+
+import home_robot.utils.pose as pu
+from home_robot.core.interfaces import Observations
+from home_robot.perception.detection.detic.detic_perception import DeticPerception
+from home_robot.mapping.semantic.categorical_2d_semantic_map_state import Categorical2DSemanticMapState
+from home_robot.mapping.semantic.categorical_2d_semantic_map_module import Categorical2DSemanticMapModule
+
+
+coco_categories = [
+    "chair",
+    "couch",
+    "potted_plant",
+    "bed",
+    "toilet",
+    "tv",
+    "dining_table",
+    "oven",
+    "sink",
+    "refrigerator",
+    "book",
+    "clock",
+    "vase",
+    "cup",
+    "bottle",
+]
+
+coco_categories_color_palette = [
```

Comment:
It's weird to me that these are just len(palette) = N*3, instead of an n x 3 matrix